Cake.DocumentDb/Requests/DocumentOperations.cs

```diff
@@ -190,18 +191,42 @@ public DocumentOperations(ClientFactory clientFactory, CollectionOperations coll
             }
         }
 
+        // https://github.com/Azure/azure-cosmos-dotnet-v2/blob/master/samples/documentdb-benchmark/Program.cs
         internal async Task PerformTask(IMigrationTask task, Action<JObject> mapAction)
         {
             var collectionResource = collectionOperations.GetOrCreateDocumentCollectionIfNotExists(
                 task.DatabaseName,
                 task.CollectionName);
 
-            var queryable = client.CreateDocumentQuery<JObject>(collectionResource.SelfLink,
-                new FeedOptions { EnableCrossPartitionQuery = true });
+            var documentQuery = client
+                .CreateDocumentQuery<JObject>(
+                    collectionResource.SelfLink,
+                    new FeedOptions { EnableCrossPartitionQuery = true })
+                .AsDocumentQuery();
 
-            var documentQuery = queryable.AsDocumentQuery();
             var isMatch = task.Filter?.Compile() ?? (doc => true);
 
+            var offer = (OfferV2)client.CreateOfferQuery()
+                .Where(o => o.ResourceLink == collectionResource.SelfLink)
+                .AsEnumerable()
+                .FirstOrDefault();
+
+            var taskCount = 5;

```

Comment:
why 5?

Reply:
Should probably be 4 as the minimum RUs currently allowable on Azure is 400 and we're allowing 1 task per 100 RUs. Will update.